### PR TITLE
Ec 204 - Incorrect stat

### PR DESCRIPTION
**Pull request by Bromel777**

### CODE REVIEW

File: src/main/scala/encry/network/EncryNodeViewSynchronizer.scala [hunk 1]
```diff
@@ -107,6 +107,8 @@ class EncryNodeViewSynchronizer(syncInfoSpec: EncrySyncInfoMessageSpec.type) ext
     case DataFromPeer(spec, data: ModifiersData@unchecked, remote) if spec.messageCode == ModifiersSpec.messageCode =>
       val typeId: ModifierTypeId = data._1
       val modifiers: Map[ModifierId, Array[Byte]] = data._2
+      if (settings.node.sendStat)
+        context.system.eventStream.publish(GetModifiers(typeId, modifiers.keys.toSeq))
```
Comment (kapinuss) [inline src/main/scala/encry/network/EncryNodeViewSynchronizer.scala]: If u wonna send message to current actor please use <<!>>

File: src/main/scala/encry/stats/StatsSender.scala [hunk 2]
```diff
@@ -5,52 +5,76 @@ import java.util
...
   influxDB.setRetentionPolicy("autogen")
 
   override def preStart(): Unit = {
-    context.system.eventStream.subscribe(self, classOf[SemanticallySuccessfulModifier[_]])
-    influxDB.write(8189, util.Arrays.asList(
-      s"nodesStartTime value=${settings.network.nodeName}"
-    ))
+    context.system.eventStream.subscribe(self, classOf[BestHeaderInChain])
+    context.system.eventStream.subscribe(self, classOf[SendDownloadRequest])
+    context.system.eventStream.subscribe(self, classOf[GetModifiers])
```
[the remark this answers is not in the document]
Comment (Bromel777) [reply src/main/scala/encry/stats/StatsSender.scala]: @kapinuss done

File: src/main/scala/encry/network/EncryNodeViewSynchronizer.scala (hunk 1 above)
Comment (Bromel777) [reply src/main/scala/encry/network/EncryNodeViewSynchronizer.scala]: @kapinuss done

File: src/main/scala/encry/stats/StatsSender.scala [hunk 3]
```diff
@@ -5,52 +5,71 @@ import java.util
...
+  var modifiersToDownload: Map[String, (ModifierTypeId, Long)] = Map()
+
   influxDB.setRetentionPolicy("autogen")
 
   override def preStart(): Unit = {
-    context.system.eventStream.subscribe(self, classOf[SemanticallySuccessfulModifier[_]])
-    influxDB.write(8189, util.Arrays.asList(
-      s"nodesStartTime value=${settings.network.nodeName}"
-    ))
+    influxDB.write(8189, s"nodesStartTime value=" + '\"' + settings.network.nodeName + '\"')
```
Comment (garmonbozzzia) [inline src/main/scala/encry/stats/StatsSender.scala]: replace by s"""..."""

Comment (Bromel777) [reply src/main/scala/encry/stats/StatsSender.scala]: @garmonbozzzia done

**Pull request merged**